assets/src/components/parts/janus-navigation-button/NavigationButton.tsx

```diff
@@ -137,9 +138,7 @@ const NavigationButton: React.FC<any> = (props) => {
     if (sSelected === undefined) {
       sSelected = currentStateSnapshot[`stage.${id}.selected`];
     }
-    if (sSelected !== undefined) {
-      setButtonSelected(sSelected);
-    }
+    setButtonSelected(parseBoolean(sSelected));
```

Comment:
why are we moving out of the `!== undefined` test for these? it should only be set if it has some value, otherwise it will be reset to false unless something specifically wants it to be true

Reply:
updated.